packages/framework/src/Model/Pricing/PriceCalculation.php

```diff
@@ -9,6 +9,8 @@
 
 class PriceCalculation
 {
+    protected const PRICE_CALCULATION_MAX_SCALE = 12;
```

Comment:
I'm wondering... why 12? please explain in the commit message

Reply:
Well, you're right to be wondering. I've changed it to 6 in bf73ba5e2, and I'll add the reason (consistency with the scale of persisted prices of products) to the commit message during rebase.